src/_pytest/config/__init__.py

```diff
@@ -1112,9 +1112,7 @@ def add_cleanup(self, func: Callable[[], None]) -> None:
     def _do_configure(self) -> None:
         assert not self._configured
         self._configured = True
-        with warnings.catch_warnings():
```

Comment:
not sure what this was for - it just disables the user specified warning filter during config. We don't want this.